### DISCUSSION TITLE
Plug-in GUI opening minimized? 1.2.0-beta 3

### DISCUSSION THREAD

**Discussion by sparksandshadows** [apohl79/audiogridder#588] [General]
When I open plug-ins in 1.2.0-beta3 it looks like this:
![image](https://user-images.githubusercontent.com/82538597/119450217-7073bd80-bce8-11eb-81d7-64ca3e7ecfbe.png)
They're sized normally in 1.1.1

**Comment by sparksandshadows** [comment]
Also to note: This happens every other time I open the plug-in.. very strange.

![image](https://user-images.githubusercontent.com/82538597/119452800-9ea6cc80-bceb-11eb-92c1-7096408f68d0.png)

**Comment by CaBuffalo** [comment]
Close it and open it again. Does it work normally?

On Mon, May 24, 2021 at 11:56 PM sparksandshadows ***@***.***>
wrote:

> Also to note: This happens every other time I open the plug-in.. very
> strange.
>
> [image: image]
> <https://user-images.githubusercontent.com/82538597/119452800-9ea6cc80-bceb-11eb-92c1-7096408f68d0.png>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/588#discussioncomment-779598>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6B4FBURT373R2AOG5TTPNC2BANCNFSM45OTPU3Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by sparksandshadows** [reply]
This is the behavior: opening a new session or plugin first time it opens in the smaller shape, second time correctly, third time smaller box, on and on alternating.

**Reply by sparksandshadows** [reply]
Mind you, this is new to 1.2.0 Beta3, 1.1.1 behaves as expected.

**Comment by CaBuffalo** [comment]
I see. I haven't experienced it in Logic Pro X.

Let's see what Andreas says about it.

On Tue, May 25, 2021 at 12:30 PM sparksandshadows ***@***.***>
wrote:

> Mind you, this is new to 1.2.0 Beta3, 1.1.1 behaves as expected.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/588#discussioncomment-782921>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6H65YSZTYSRZ72HZZ3TPP3GRANCNFSM45OTPU3Q>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by igbro** [comment]
I got the same Using VST3 in Cubase pro 11 (W10).

It's happening once every two tries. Not happening with VST2 instances.

I already reported this to Andreas a few days ago.

https://user-images.githubusercontent.com/65025818/119673952-171ca300-be3c-11eb-921e-648eea400876.mp4

Here is the video I sent to Andreas showing VST2 working and VST 3 failing on 1st... 2d... 4th attempt.

**Reply by sparksandshadows** [reply]
Exact same behavior over here. Yup.
